CONTRIBUTING.md

```diff
@@ -2,36 +2,38 @@
+
+## Ongoing Initiatives
+
+We use Github Discussion to organise our discussion along new intiatives and organise ourselves. Please do check the [announcements](https://github.com/recharts/recharts/discussions/categories/announcements) section for an overview of recent initiatives. 
 
 ## <a name="issues"></a>Issues and Bugs
 
 ### Where to Find Known Issues
 
-We will be using [GitHub Issues](https://github.com/recharts/recharts/issues) for our public bugs. We will keep a close eye on this and try to make it clear when we have an internal fix in progress. Before filing a new task, try to make sure your problem doesn't already exist.
+We will be using [GitHub Issues](https://github.com/recharts/recharts/issues) for our bugs and feature requests. We will keep a close eye on this and try to make it clear when we have an internal fix in progress. Before filing a new task, try to make sure your problem doesn't already exist.
 
 ### Reporting New Issues
 
-The best way to get your bug fixed is to provide a reduced test case. jsFiddle  provide a way to give live examples. You can fork our example in [recharts.org](http://recharts.org/) to show your case.
-
+The best way to get your bug fixed is to provide a reduced test case. jsFiddle provide a way to give live examples. You can fork our example in [recharts.org](http://recharts.org/) to show your case.
 
 ## <a name="pr"></a>Pull Requests
 
-**Working on your first Pull Request?** You can learn how from this *free* series [How to Contribute to an Open Source Project on GitHub](https://app.egghead.io/playlists/how-to-contribute-to-an-open-source-project-on-github)
+**Working on your first Pull Request?** You can learn how from this _free_ series [How to Contribute to an Open Source Project on GitHub](https://app.egghead.io/playlists/how-to-contribute-to-an-open-source-project-on-github)
 
-*Before* submitting a pull request, please make sure the following is done…
+_Before_ submitting a pull request, please make sure the following is done…
 
-+ Search [GitHub](https://github.com/recharts/recharts/pulls) for an open or closed Pull Request that relates to your submission. You don't want to duplicate effort.
-
-+ Fork the repo and create your branch from `master`.
-+ If you've added code that should be **tested**, add tests!
-+ If you've changed APIs, update the [documentation](https://github.com/recharts/recharts.org) and demo.
-+ Ensure the test suite passes (`npm run test`).
-+ Make sure your code lints (`npm run lint`) - we've done our best to make sure these rules match our internal linting guidelines.
+- Search [GitHub](https://github.com/recharts/recharts/pulls) for an open or closed Pull Request that relates to your submission. You don't want to duplicate effort.
 
+- Fork the repo and create your branch from `master`.
+- If you have added functionality or changed existing functionality, be sure to add a test. Ideally a unit test for helper function, or a test that includes rendering with RTL.
```

Comment:
Could a link be added to an example test and how to run it? Sometimes it difficult to figure out how to run a test in isolation and what a good test looks like for a new project.

Reply:
I added a whole section on testing. Thank you, good pointer!